text/java/0003-clojure-tools.md

```diff
@@ -0,0 +1,33 @@
+# Clojure Tools Buildpack
```

Comment:
Can you rename this file to `0000-clojure-tools.md`? We usually keep the number as 0000 until it's approved, then it gets a number assigned at approval time.